eclass/gkrellm-plugin.eclass

```diff
@@ -96,6 +107,6 @@ gkrellm-plugin_src_install() {
 		fi
 	fi
 }
-
```

Comment:
put the one empty vertical whitespace line back

Reply:
Done.